ci/travis-37.yaml

```diff
@@ -0,0 +1,14 @@
+name: pandas
+channels:
+  - defaults
+  - conda-forge
+  - c3i_test
```

Comment:
@mingwandroid do you know if `c3i_test` will keep these 3.7 packages around until they're in the defaults?